File: Extras/Guia Grafos/ej18Mir.py
def encontrarCamino(x, y, anterior, distancia):
	if x == y :
		return distancia

	izq = x - 1
	der = 2 * x

	if abs(y-izq) < abs(y-x):
		distIzq = encontrarCamino(izq, y, x, distancia+1) 
	if abs(y-der) < abs(y-x):
		distDer = encontrarCamino(der, y, x, distancia+1)

	if distIzq < distDer:
		return distIzq
        
	return distDer

File: Extras/Guia Grafos/ej18Mir.go
func encontrarCamino(x, y int, distancia int) int {
	if x == y {
		return distancia
	}

	izq := x - 1
	der := 2 * x

	distIzq := encontrarCamino(izq, y, distancia+1)
	distDer := encontrarCamino(der, y, distancia+1)

	if distIzq < distDer {
		return distIzq
	}
	return distDer

}
